app/forms/GoodsDeparted.scala

```diff
@@ -29,8 +30,8 @@ object GoodsDeparted {
   val formId = "GoodsDeparted"
 
   object AllowedPlaces {
-    val outOfTheUk = "OutOfTheUk"
-    val backIntoTheUk = "BackIntoTheUk"
+    val outOfTheUk = "outOfTheUk"
```

Comment:
controversial

Reply:
Why?   The actual values aren't submitted anywhere and I wanted to re-use them to create message keys (at one point - not doing that any more so I could revert the above change if it upsets anyone).